MaxiOps/CCTP_Bridge/receive/claimArbiUSDC.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1715193245588,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.5",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x383bbde1a0fa18089fef4d0f07fca15a785b14563241a3e34530459e2fdddc97"
+  },
+  "transactions": [
+    {
+      "to": "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "name": "message", "type": "bytes", "internalType": "bytes" },
+          { "name": "attestation", "type": "bytes", "internalType": "bytes" }
+        ],
+        "name": "receiveMessage",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "message": "0x000000000000000000000003000000000000D6E2000000000000000000000000BD3FA81B58BA92A82136038B25ADEC7066AF315500000000000000000000000019330D10D9CC8751218EAF51E8885D058642E08A000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000A0B86991C6218B36C1D19D4A2E9EB0CE3606EB48000000000000000000000000C38C5F97B34E175FFD35407FC91A937300E3386000000000000000000000000000000000000000000000000000000000BEBC2000000000000000000000000000C38C5F97B34E175FFD35407FC91A937300E33860",
```

Comment:
To get the message we have to look at the events of teh send transaction:
https://etherscan.io/tx/0x3203fa6e0f6bfe9f78b7179f2d8611168b5488e7e6e870f645cd7e27920b50ec#eventlog

In this case the Arbi bridge is the first brridge in the batch so we are looking for the first message sent transaction.  Then we pull the value from there and put a 0x in front to get message:
<img width="1279" alt="image" src="https://github.com/BalancerMaxis/multisig-ops/assets/11885129/8f8aebeb-c787-4dd8-bd71-9a798094ec84">